homeassistant/components/repetier/sensor.py

```diff
@@ -191,7 +191,7 @@ def update(self):
         job_name = data["job_name"]
         start = data["start"]
         from_start = data["from_start"]
-        self._state = datetime.utcfromtimestamp(start)
+        self._state = dt_util.utc_from_timestamp(start).replace(tzinfo=None)
```

Comment:
Unsure if we could work with aware datetime here.

Reply:
same

Reply:
We should fix it here too as we fixed it above, right?

Reply:
Yes! Done.